.changes/unreleased/Dependencies-20240331-103917.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Dependencies
+body: Remove unwanted dependency of protobuf in dev-requirements
```

Comment:
Small nit: 
Can we replace the word `unwanted` with `duplicate` here. The word `unwanted` without the context that this dependency is getting pulled in somewhere else makes it sound like we're dropping the dependency completely.